### ISSUE TITLE
Module 4

### ISSUE THREAD

**Issue by azharkhuwaja** [MicrosoftLearning/AZ-304-Microsoft-Azure-Architect-Design#90]
# Module: 04
#### Step: 13

Step 13 of Task 04 is about deleting tenant "Adatum Lab". But a tenant can't be deleted unless you cancel all its subscriptions. There are certain steps required to cancel the subscription which are missing in this step. It will take either 30 days (if we don't do anything) or 3 days (if we cancel trial subscription) to enable us to delete the tenant. That information is missing from this step. Thanks

**Comment by azharkhuwaja** [comment]
any update? thanks

**Comment by staleycyn** [comment]
No one else has reported a problem with this. If you have the steps submit a Pull Request so it can be reviewed.

**Comment by shivams619** [comment]
> # Module: 04
> ## Lab/Demo: 03
> ### Task: 04
> #### Step: 13
> Step 13 of Task 04 is about deleting tenant "Adatum Lab". But a tenant can't be deleted unless you cancel all its subscriptions. There are certain steps required to cancel the subscription which are missing in this step. It will take either 30 days (if we let trial expire) or 3 days (if we cancel subscription of Azure AD Premium P2 Trial) to enable us to delete the tenant. That information is missing from this step. Thanks

Yeah I have faced this issue many times. I just request the participants to move on without deleting.

**Comment by azharkhuwaja** [comment]
I have modified the instructions to resolve this issue for my students via a local word file. I am not familiar with procedure to Pull Request. Thanks
